### PR TITLE
Init cluster/build05

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @openshift/test-platform

### CODE REVIEW

File: clusters/build-clusters/build05/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+* the SSH key pair (`id_rsa` and `id_rsa.pub`): BitWarden item `build_farm_build02`.
+* The auth info for `kubeadmin` and the cert-based kubeconfig file (attachment `b05.admin.cert.kubeconfig`): BitWarden item `build_farm_build05`. (TODO: hongkliu)
+
+## Installation
+
+### GCP Project
+The gcp project `openshift-ci-build-farm` for installation of this cluster: [gcp console](https://console.cloud.google.com/home/dashboard?project=openshift-ci-build-farm). The project is created by [DPP team](hhttps://issues.redhat.com/browse/DPP-4926).
+
+### Base domain, Service account and Public Key
+We reused the public zone "gcp.ci.openshift.org" for [build02](../02_cluster) which is the base domain for installer, and the service account `ocp-cluster-installer`.
```
Comment (petr-muller) [inline clusters/build-clusters/build05/README.md:17]:
```suggestion
### Base domain, Service account and Public Key
We reused the public zone "gcp.ci.openshift.org" for [build05](../05_cluster) which is the base domain for installer, and the service account `ocp-cluster-installer`.
```

File: clusters/build-clusters/build05/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Download the attachment "openshift-ci-build-farm-64e4ce412ae3.json" and `id_rsa.pub` of BW item `build_farm_build02`.
```
Comment (petr-muller) [inline clusters/build-clusters/build05/README.md:19]:
```suggestion
Download the attachment "openshift-ci-build-farm-64e4ce412ae3.json" and `id_rsa.pub` of BW item `build_farm_build05`.
```

File: clusters/build-clusters/build05/README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+### Region
+
+`build02` and `build04`: `us-east1`.
```
Comment (petr-muller) [inline clusters/build-clusters/build05/README.md:23]: Do we need to mentiopn b02 and b04 in a b05 README?

**Comment by hongkailiu** [comment]
/hold

We might need no long this PR, expecting b02 to recover.

**Pull request closed without merging**